README.md

```diff
@@ -1,2 +1,27 @@
-# Animated-navbar
-Whathevs
\ No newline at end of file
+# Toolbar icons animation challenge
+(Add image)
+
+## Description
+The aim of this project is to practice HTML, CSS and Vanilla JS by building a toolbar icons animation.
+This challenge helped to realize how we can build sophisticated things with simple code.
+
+## Installation
+Clone this repository to get a version in your local environment.
+- On GitHub, navigate to the main page of the repository,
+- Above the list of files, click the green buttton (Code),
+- To clone the repository using HTTPS, copy the link starting with "HTTPS",
+- Open Git Bash,
+- Change the current working directory to the location where you want the cloned repository,
+- Type git clone and then paste the URL you copied earlier,
+- Press enter to create a local clone.
+
+## Demo
```

Comment:
Uh demo is a nice idea! Do you know how to publish a website from GitHub? There are 2 popular ways:
- Github Pages
- Netlify

Personally I prefer [Netlify](https://www.netlify.com/). It's more [friendly and powerful](https://www.aleksandrhovhannisyan.com/blog/github-pages-vs-netlify/) and it supports deploys from PR. Give it a try!

Reply:
I planned to insert the GIF in the README.md but I can do both If you think that I could be a good exercice for me.

Reply:
Yes, try to integrate Netlify on this github repo. When integrated, a new deploy should run on this PR after each commit pushed.